Foundation/FileManager.swift

```diff
@@ -704,43 +856,41 @@ open class FileManager : NSObject {
      
         This method replaces fileSystemAttributesAtPath:.
      */
- #if os(Android)
-    @available(*, unavailable, message: "Unsuppported on this platform")
     open func attributesOfFileSystem(forPath path: String) throws -> [FileAttributeKey : Any] {
-        NSUnsupported()
```

Comment:
Its better to leave in the `@available` for Android as it will catch usage at compile time  rather than the `NSUnsupported()` below,  at runtime.

Reply:
You are right

Reply:
I still wonder why it is unsupported on Android!